test/api/assignments_test.rb

```diff
@@ -72,8 +74,9 @@ def test_provides_a_useful_error_message_when_key_is_wrong
   def test_api_accepts_submission_on_completed_exercise
     Notify.stub(:everyone, nil) do
       Xapi.stub(:exists?, true) do
+        solution = { "THE CODE" => "go/one/code.go" }
         post '/user/assignments', { key: alice.key, code: 'THE CODE', path: 'go/one/code.go', 
-          'language' => 'go', 'problem' => 'one' }.to_json
+          'language' => 'go', 'problem' => 'one', solution: solution }.to_json
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.